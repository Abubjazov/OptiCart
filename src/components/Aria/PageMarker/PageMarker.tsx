export const PageMarker = ({ pageName }: PageMarkerProps): JSX.Element => (
	<p
		role='alert'
		style={{
			display: 'block',
			position: 'fixed',
			left: 100,
			top: 0,
			overflow: 'hidden',
			height: 0,
			opacity: 1,
		}}
	>
		You are on the {pageName} page
	</p>
)

export interface PageMarkerProps {
	pageName: string
}
