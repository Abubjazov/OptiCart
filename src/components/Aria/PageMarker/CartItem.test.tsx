import { render } from '@testing-library/react'
import { PageMarker } from './PageMarker'

describe('Component: PageMarker', () => {
	test('should render PageMarker', () => {
		const { asFragment } = render(<PageMarker pageName='test' />)

		expect(asFragment()).toMatchSnapshot()
	})
})
