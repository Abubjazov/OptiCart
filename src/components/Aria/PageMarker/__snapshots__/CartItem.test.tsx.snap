// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component: PageMarker should render PageMarker 1`] = `
<DocumentFragment>
  <p
    role="alert"
    style="display: block; position: fixed; left: 100px; top: 0px; overflow: hidden; height: 0px; opacity: 1;"
  >
    You are on the test page
  </p>
</DocumentFragment>
`;
